source/retryable-reads/tests/README.rst

```diff
@@ -171,3 +174,5 @@ Changelog
              now expressed within ``runOn`` elements.
 
              Add test-level ``useMultipleMongoses`` field.
+
+:2020-06-12: Add the default value for heartbeatFrequencyMS.
```

Comment:
This change description doesn't match up with the diff. I'd suggest:

> Suggest lowering heartbeatFrequencyMS in addition to minHeartbeatFrequencyMS

And please update the date.